#Header header
string text_to_say

# not used at the moment, but maybe some time in the future
# we may want to add a decay time in the future
#time decay_time
